introduce PARALLEL_TESTS for builder_run_tests (all builders) (#446)
ros-industrial/industrial_ci: doc/index.rst

* **NOT_TEST_DOWNSTREAM** (default: not set): If true, tests in the downstream workspace won't be run.
* **OS_CODE_NAME** (default: derived from ROS_DISTRO): See `this section for the detail <https://github.com/ros-industrial/industrial_ci/blob/master/doc/index.rst#optional-type-of-os-and-distribution>`__.
* **OS_NAME** (default: ubuntu): Possible options: {`ubuntu`, `debian`}. See `this section for the detail <https://github.com/ros-industrial/industrial_ci/blob/master/doc/index.rst#optional-type-of-os-and-distribution>`__.

> * **PARALLEL_TESTS** (default: not set): If `false`, build tool (e.g. Catkin) executes `run_tests` step sequentially, i.e. only one job at a time (equal to `-j1`).

Reviewer: ```suggestion
* **PARALLEL_TESTS** (default: not set): If `true`, build tool (e.g. Catkin) executes `run_tests` jobs in parallel, instead of only one job at a time.
```
